wagtail_localize/views/update_translations.py

```diff
@@ -17,16 +17,35 @@
 from wagtail.snippets.models import get_snippet_models
 from wagtail.utils.version import get_main_version
 
+from wagtail_localize.machine_translators import get_machine_translator
 from wagtail_localize.models import TranslationSource
+from wagtail_localize.views.edit_translation import apply_machine_translation
 from wagtail_localize.views.submit_translations import TranslationComponentManager
 
 
+if get_machine_translator():
+    PUBLISH_TRANSLATIONS_HELP = (
+        "Select this to publish immediately. Changes will be published "
+        "in the original language unless you also select use machine translation."
+    )
+else:
+    PUBLISH_TRANSLATIONS_HELP = (
+        "Select this to publish immediately. Changes will be published "
+        "in the original language until you translate and publish them."
+    )
```

Comment:
I think these need tweaking as there is a discrepancy in style between this and the use machine translation help.

I suggest something along the lines of:

> Apply the updates and publish immediately. The changes will use the original language until translated unless you also select "Use machine translation".

and 

> Apply the updates and publish immediately. The changes will use the original language until translated.

Reply:
Thank you, I've used your copy verbatim.